_includes/pbc/all/install-features/202304.0/install-glue-api/install-backend-api-glue-json-api-convention.md

```diff
@@ -0,0 +1,141 @@
+
+
+
+This document describes how to integrate the Glue JSON:API convention for Backend API application into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Glue JSON:API convention core.
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME          | VERSION   | INTEGRATION GUIDE  |
+|---------------|-----------|--------------------|
+| Spryker Core  | Feature   | {{page.version}}   |
```

Comment:
The integration guide has to have the following value:
* [Spryker core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html)